igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/model/GoogleCloudStorageObject.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.gcb.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+public class GoogleCloudStorageObject {
```

Comment:
Kind of surprised there's no helper class for this in the client lib.

Reply:
Yeah, I think it's because the client is just a generated client for the REST APIs which only understand `bucket` and `object` syntax, whereas the `gs://bucket/object` is handled by `gsutil`.  There is a newer, more idiomatic library to supersede the generated one, but we're using the generated one everywhere else (and the new one doesn't support GCB yet)...maybe that one supports this, but I didn't want to bring it in just for this, so I didn't check.